addons/survey/wizard/survey_invite_views.xml

```diff
@@ -5,27 +5,30 @@
             <field name="name">survey.invite.view.form</field>
             <field name="model">survey.invite</field>
             <field name="arch" type="xml">
-                <form string="Compose Email" class="o_mail_composer_form">
+                <form string="Compose Email" class="o_mail_composer_form" style="padding-bottom: 0px;">
                     <group col="1">
                         <group col="2">
                             <field name="survey_access_mode" invisible="1"/>
                             <field name="survey_users_login_required" invisible="1"/>
                             <field name="survey_users_can_signup" invisible="1"/>
-                            <field name="survey_id" readonly="context.get('default_survey_id')"/>
+                            <field name="survey_id" readonly="context.get('default_survey_id')" invisible="1"/>
                             <field name="existing_mode" widget="radio" invisible="1" />
                             <field name="render_model" invisible="1"/>
-                            <field name="survey_start_url" label="Public share URL" readonly="1" widget="CopyClipboardChar"
-                                 attrs="{'invisible':[('survey_access_mode', '!=', 'public')]}"
-                                 class="mb16"/>
+                            <label class="o_survey_label_survey_start_url" for="survey_start_url" string="Survey Link"/>
```

Comment:
You need to add the same attrs as the field (or wrap in a div that has the attrs to avoid duplication).

Currently it shows like this (oopsie ^^):

![image](https://user-images.githubusercontent.com/41282536/202490384-10fdc16f-d8b0-4b33-b1e8-63e8517fd127.png)